### PR TITLE
Header fixed during scroll

### PR DESCRIPTION
**Pull request by sofipacifico**
* Added library for to fix header during scroll
* Added border when the input is focused

### CODE REVIEW

**Comment by sofipacifico** [comment]
image

**Comment by sofipacifico** [comment]
@AlphaGit @andresmoschini @ragojose @Gkolocsar

**Comment by sofipacifico** [comment]
image

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml [hunk 1]
```diff
@@ -76,9 +77,9 @@
         <!-- /ko -->
     </div>
     <table data-bind="attr: {class: numberOfColumns}, css: {table: true, 'calification-items': true}">
-        <thead>
+        <thead id="header-fixed">
             <tr>
-                <th class="calificators"></th>
+                <th class="calificators"><h1>EVALUADORES</h1></th>
```
Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: Use CSS to make this uppercase.

**Comment by AlphaGit** [comment]
I was going to suggest using `position: sticky`, but it seems it is not that widespread or accepted yet: http://caniuse.com/#feat=css-sticky What do you think? @sofipacifico @andresmoschini @Gkolocsar

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml (hunk 1 above)
Comment (AlphaGit) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: (That is, if you really want to make it uppercase)

**Comment by sofipacifico** [comment]
Sound good, but looks like is not supported by the all navigators

File: source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml (hunk 1 above)
Comment (sofipacifico) [inline source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml]: :+1:

**Comment by Gkolocsar** [comment]
It seems that position: sticky is not widely accepted yet. Might be an option in the future though.

**Comment by Gkolocsar** [comment]
This is a really nice improvement on the califications page. I'm not that experienced in front-end to make any more constructive suggestions, but it's looking good! :)

**Comment by sofipacifico** [comment]
@AlphaGit sounds good for you?

**Comment by AlphaGit** [comment]
Yup, looks good to me!

**Comment by sofipacifico** [comment]
:+1:  Merged

**Pull request merged**